Cargo.toml

```diff
@@ -2,15 +2,15 @@
 name = "projectm_sdl"
 version = "0.1.0"
 edition = "2021"
+rust-version = "1.68.2"
 
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
 libc = "*"
-sdl2 = "0.35.2"
+sdl2 = "0.35"
 # projectm-rs = "^1.0.4"
-projectm-rs = { path = "../projectm-rs" }
-clippy = "0.0.302"
+projectm-rs = { git = "https://github.com/projectM-visualizer/projectm-rs" }
```

Comment:
I'd prefer we pull from crates.io instead of git so its always locked to a working version and not to our development (unless this is your goal temporarily).

projectm-rs = { version = "1.0.5" }

Reply:
Done.